src/interfaces/events-subscribe.ts

```diff
@@ -0,0 +1,58 @@
+import type { Signer } from '../types/signer.js';
+import type { EventsFilter, EventsSubscribeDescriptor, EventsSubscribeMessage } from '../types/events-types.js';
+
+import { AbstractMessage } from '../core/abstract-message.js';
+import { Message } from '../core/message.js';
+import { removeUndefinedProperties } from '../utils/object.js';
+import { Time } from '../utils/time.js';
+import { DwnInterfaceName, DwnMethodName } from '../enums/dwn-interface-method.js';
+
+
+export type EventsSubscribeOptions = {
+  signer: Signer;
+  messageTimestamp?: string;
+  filters?: EventsFilter[]
+  permissionsGrantId?: string;
+};
+
+export class EventsSubscribe extends AbstractMessage<EventsSubscribeMessage> {
+  public static async parse(message: EventsSubscribeMessage): Promise<EventsSubscribe> {
+    Message.validateJsonSchema(message);
+    await Message.validateSignatureStructure(message.authorization.signature, message.descriptor);
+
```

Comment:
Sanity: should we do more here like other similar interface methods, such as validate structure of the filter?